azure/iac/web.rego

```diff
@@ -63,7 +63,7 @@ https_only_metadata := {
     "Type": "IaC",
     "Product": "AZR",
     "Language": "ARM template",
-    "Policy Title": "Ensure Azure App Service Web App enforce https connection",
+    "Policy Title": "Ensure Azure Web Service Web App enforce https connection",
```

Comment:
@shahinshirvani  existing message was correct i believe. web app is an Azure App service.